### PR TITLE
KUZ-512 : Add updateSelf action's doc

### PR DESCRIPTION
**Pull request by dbengsch**

### CODE REVIEW

File: source/includes/_authController.md [hunk 1]
```diff
@@ -284,3 +284,62 @@ The **_logout** action doesn't take strategy.
...
+      ....
+    }
+  }
+}
+```
+
+Updates the current User object in Kuzzle's database layer.
+<aside class="notice">
+User must be connected in order to perform this action.
+The current user can not update its profile using this action. [Security / updateUser](#updateuser) must be used instead.
```
Comment (stafyniaksacha) [inline source/includes/_authController.md]: Inside a html tag (`<aside>` here) markdown are not parsed: 

Change:
`[Security / updateUser](#updateuser)` 
By 
`<a href="#updateuser">Security / updateUser</a>`

**Comment by scottinet** [comment]
Apart from @stafyniaksacha comment: +1

File: source/includes/_authController.md (hunk 1 above)
Comment (dbengsch) [inline source/includes/_authController.md]: fixed

**Comment by jenow** [comment]
+1

**Comment by ballinette** [comment]
+1

**Pull request merged**